UnboundLib/Patches/GM_Test.cs

```diff
@@ -36,19 +36,11 @@ static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> inst
 
             for (int i = 0; i < list.Count; i++)
             {
-                if (
-                    list[i].LoadsField(f_pmInstance) &&
-                    list[i + 2].OperandIs(m_playerDied) &&
-                    list[i + 4].Calls(m_addPlayerDied)
-                )
+                if (list[i].LoadsField(f_pmInstance) && list[i + 2].OperandIs(m_playerDied) && list[i + 4].Calls(m_addPlayerDied))
```

Comment:
I think a lot of these changes come from people (Dan in this case) applying their IDE's automatic formatting blindly to the entire project and then blindly using `git commit -a` instead of just committing their changes.

Reply:
Yeah most of my stuff is done pretty late in the night as it's my only free time from work related stuff, I end up just missing out on a few of these changes, I always try to revert these back when it happens

Reply:
plug for `git add -p` 💯